### PR TITLE
Enhance ocean conservation task

### PR DESCRIPTION
**Pull request by cbegeman**
Here, we add conservation plot types for all the ocean's mass flux terms.

### CODE REVIEW

**Comment by cbegeman** [comment]
### Testing

This analysis has been run on chrysalis including against a control run: https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.cbegeman/analysis/20240611.HRr5_test9.chrysalis/yrs1-10_clim1-10_conserv/ocean/index.html

File: mpas_analysis/shared/io/mpas_reader.py [hunk 1]
```diff
@@ -194,7 +194,10 @@ def _parse_dataset_time(ds, inTimeVariableName, calendar,
         # convert to string
         timeStrings = [''.join(xtime.astype('U')).strip()
                        for xtime in timeVar.values]
-        days = string_to_days_since_date(dateString=timeStrings,
+        for i, timeString in enumerate(timeStrings):
+             if timeString[8:] == '15_00:00:00':
+                 timeStrings[i] = f'{timeStrings[i+1][:7]}-01_00:00:00'
+             days = string_to_days_since_date(dateString=timeStrings,
```
Comment (xylar) [inline mpas_analysis/shared/io/mpas_reader.py:200]: This didn't work for me.  Should this be:
```suggestion
        for i, timeString in enumerate(timeStrings):
             if timeString[8:] == '15_00:00:00':
                 timeStrings[i] = f'{timeStrings[i+1][:7]}-01_00:00:00'
        days = string_to_days_since_date(dateString=timeStrings,
```

**Comment by cbegeman** [comment]
@xylar Would you like to try running again with the time string fix? I think your case might be more likely to trigger an issue.

**Review by xylar — APPROVED** [review]
## Testing

I reran the test suite after rebasing this branch onto #1010 (which includes some fixes unrelated to this branch).  Results are here:
https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.xasay-davis/analysis_testing/chrysalis/enhance-ocn-conservation-task/

Everything looks good, notably the sea-ice time series are now plotting without error.

**Comment by cbegeman** [comment]
Thank you, @xylar!

**Pull request merged**